app/models/settings/general.rb

```diff
@@ -93,9 +93,6 @@ class General < Base
     }
     setting :twitter_hashtag, type: :string
 
-    # Sponsors
-    setting :sponsor_headline, default: -> { I18n.t("models.settings.general.community_sponsors") }
```

Comment:
The headline was being used for the sponsorship widget in the sidebar which I've deleted.